tests/Tests/E2e/ExampleE2eTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenEMR\Tests\E2e;
+
+use Symfony\Component\Panther\PantherTestCase;
+use Symfony\Component\Panther\Client;
+
+class ExampleE2eTest extends PantherTestCase
+{
+    /**
+     * The base url used for e2e (end to end) browser testing.
+     */
+    private $e2eBaseUrl;
+
+    protected function setUp(): void
+    {
+        $this->e2eBaseUrl = getenv("OPENEMR_BASE_URL", true) ?: "http://localhost";
```

Comment:
note that when you rebase/merge in current codebase, this variable is now (as of last night :) ): 
OPENEMR_BASE_URL_E2E